sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -24,8 +26,9 @@
 @RequestMapping("/orchestration")
 class OrchestrationController {
 
-  private final OrchestrationClient client =
-      new OrchestrationClient().withLlmConfig(OpenAiModel.GPT_35_TURBO);
+  private final OrchestrationClient client = new OrchestrationClient();
+  private final OrchestrationConfig config =
+      new OrchestrationConfig().withLlmConfig(OpenAiModel.GPT_35_TURBO);
```

Comment:
users have to maintain two objects now

Reply:
I guess there must be good reasons to split the client and config.
Eg., you can reuse the client with different config.
If so, keeping two objects is fine.

Reply:
You can use the same client with different config objects, but the reason this is now two objects instead of one is that the implementation is simpler